crates/openvino/tests/classify-alexnet.rs

```diff
@@ -1,10 +1,10 @@
-//! Demonstrates using `openvino-rs` to classify an image using an AlexNet model and a prepared input tensor. See
-//! [README](fixtures/alexnet/README.md) for details on how this test fixture was prepared.
+//! Demonstrates using `openvino-rs` to classify an image using an AlexNet model and a prepared
+//! input tensor. See [`fixtures/alexnet`] for details on how this test fixture was prepared.
```

Comment:
`fixtures/alexnet` doesn't have details anymore, they're moved to `mod.rs` comments. Same for `classify-mobilenet.rs` and `classify-inception.rs`. I do see the Readme present in the remote location.

Reply:
I just decided to remove all of that; if anyone wants to know more, they should navigate to the `fixtures::alexnet` module and there's some documentation there they can use.